daily_word_bot/app.py

```diff
@@ -102,6 +107,61 @@ def on_get_blockwords_callback(self, update: Update, context: CallbackContext, i
         else:
             update.message.reply_text(msg, reply_markup=reply_markup)
 
+    def on_mylevels_callback(self, update: Update, context: CallbackContext, is_inline_keyboard=False) -> None:  # pragma: no cover
+        # get user information from the message
+        message = update.message or update.callback_query.message
+        chat_id = message.chat_id
+
+        # look for the levels of the user in the db
+        levels = self.dao.get_user_levels(chat_id)
+
+        # generate answer message and the markup
+        answer = utils.build_levels_answer(levels)
+
+        # answer the user
+        if is_inline_keyboard:
+            update.callback_query.edit_message_text(answer.get('msg'), reply_markup=answer.get('reply_markup'))
+        else:
+            update.message.reply_text(answer.get('msg'), reply_markup=answer.get('reply_markup'))
+
+    def on_removelevel_callback(self, update: Update, context: CallbackContext, is_inline_keyboard=False, level_to_remove='') -> None:  # pragma: no cover
+        # get user information from the message
+        message = update.message or update.callback_query.message
+        chat_id = message.chat_id
+
+        self.dao.remove_user_level(chat_id, level_to_remove)
```

Comment:
from this line you could actually return `on_mylevels_callback` passing the update & context method since the rest of the method's behavior is the same